hack/update-codegen.sh

```diff
@@ -26,5 +26,6 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 kube::golang::setup_env
 
 "${KUBE_ROOT}/hack/build-go.sh" cmd/libs/go2idl/set-gen
+"${KUBE_ROOT}/hack/build-go.sh" cmd/libs/go2idl/deepcopy-gen
```

Comment:
All on one line should work (or separate with \ if separate lines is more readable):

`"${KUBE_ROOT}/hack/build-go.sh" cmd/libs/go2idl/set-gen cmd/libs/go2idl/deepcopy-gen`